Library/Formula/libspnav.rb

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
No, It's cool now. There was an empty line at the start of the file, and apparently when that was fixed Git decided to grab the next empty line and apply my comment there.